src/admin/index.js

```diff
@@ -428,38 +431,43 @@ module.exports = ({
 
   /**
    * @param {string} groupId
-   * @param {string[]]} topics
+   * @param {string[]} topics
    * @param {boolean} [resolveOffsets=false]
    * @return {Promise}
    */
-  const fetchConsumerGroupOffsets = async ({ groupId, topics, resolveOffsets = false }) => {
+  const fetchConsumerGroupOffsets = async ({ groupId, topics = [], resolveOffsets = false }) => {
     if (!groupId) {
       throw new KafkaJSNonRetriableError(`Invalid groupId ${groupId}`)
     }
 
+    if (!Array.isArray(topics)) {
+      throw new KafkaJSNonRetriableError(`Expected topics array, got ${topics}`)
+    }
+
     const coordinator = await cluster.findGroupCoordinator({ groupId })
 
     let consumerOffsets
     if (topics.length) {
-      const topicsToFetch = Promise.all(
+      const topicsToFetch = await Promise.all(
         topics.map(async topic => {
           const partitions = await findTopicPartitions(cluster, topic)
           const partitionsToFetch = partitions.map(partition => ({ partition }))
           return { topic, partitions: partitionsToFetch }
         })
       )
+
       const { responses } = await coordinator.offsetFetch({
         groupId,
-        topicsToFetch,
+        topics: topicsToFetch,
       })
       consumerOffsets = responses
     } else {
-      const { responses } = await coordinator.offsetFetch({ groupId })
+      const { responses } = await coordinator.offsetFetch({ groupId }, [])
```

Comment:
What's with the empty array here?

Reply:
It needs to be passed to `offsetFetch` so that it will fetch all topics (it actually sets it in the protocol it sends to Kafka from what I've seen).